### PR TITLE
fix(ui): Show all projects even if not member

### PR DESCRIPTION
**Pull request by mikellykels**
When creating an alert rule, display all projects, but disable the projects the user is not a part of.

cc @robinrendle @adhiraj for design feedback

FIXES [WOR-826](https://getsentry.atlassian.net/browse/WOR-826)

### Before
Screen Shot 2021-04-19 at 3 11 01 PM

### After
Screen Shot 2021-04-21 at 3 40 56 AM

### CODE REVIEW

File: static/app/components/contextPickerModal.tsx [hunk 1]
```diff
@@ -253,10 +270,25 @@ class ContextPickerModal extends React.Component<Props> {
...
   renderProjectSelectOrMessage() {
     const {organization, projects} = this.props;
-    // only show projects the user is a part of
-    const memberProjects = projects.filter(project => project.isMember);
 
-    const projectOptions = memberProjects.map(({slug}) => ({label: slug, value: slug}));
+    const projectOptions = [
+      {
+        label: t('Projects I belong to'),
+        options: this.getMemberProjects()[0].map(p => ({
```
Comment (scttcper) [inline static/app/components/contextPickerModal.tsx:277]: lets call `getMemberProjects` once

```ts
const [memberProjects, nonMemberProjects] = this.getMemberProjects()
```

File: static/app/components/contextPickerModal.tsx [hunk 2]
```diff
@@ -193,6 +199,17 @@ class ContextPickerModal extends React.Component<Props> {
     this.navigateIfFinish([{slug: organization}], [{slug: value}]);
   };
 
+  getMemberProjects = () => {
+    const {projects} = this.props;
+    const nonMemberProjects: Project[] = [];
+    const memberProjects: Project[] = [];
+    projects.map(project =>
```
Comment (scttcper) [inline static/app/components/contextPickerModal.tsx:206]:
```suggestion
    projects.forEach(project =>
```

File: static/app/components/contextPickerModal.tsx [hunk 3]
```diff
@@ -270,22 +271,24 @@ class ContextPickerModal extends React.Component<Props> {
...
         })),
       },
       {
         label: t("Projects I don't belong to"),
-        options: this.getMemberProjects()[1].map(p => ({
+        options: nonMemberProjects.map(p => ({
           value: p.slug,
           label: t(`${p.slug}`),
-          isDisabled: true,
+          isDisabled: isSuperuser ? false : true,
```
Comment (scttcper) [inline static/app/components/contextPickerModal.tsx:291]: 👍

**Review by scttcper — APPROVED** [review]

**Comment by mikellykels** [comment]
Figured out what was wrong with the acceptance test and added a fix for it 😅

**Pull request merged**